docs/design/coreclr/profiling/davbr-blog-archive/Attach.md

```diff
@@ -96,12 +96,12 @@ So here's the catch.  What if a V4 app starts up in background GC mode _without_
 
 Of course, you could forcibly turn off concurrent / background mode every time the app starts up via a config file:
 
-| 
+|
 
-\<configuration\>   
-    \<runtime\>   
-        \<gcConcurrent enabled="false"/\>   
-    \</runtime\>   
+\<configuration\>
+    \<runtime\>
+        \<gcConcurrent enabled="false"/\>
+    \</runtime\>
```

Comment:
This have **undesired** visual change.

@stephentoub, this is an example of a problem that trailing whitespace option in editorconfig can cause in the absence of a markdownlint.

Reply:
This is poorly formatted both before and after this change. This should use ```xml instead.